requirements.txt

```diff
@@ -14,7 +14,6 @@ numpy==1.22.2
 owlready2
 pandas<1.3.0
 pyarrow>=1.0
-psutil
```

Comment:
This is creating a conflict when creating the makefile. Not quite sure why this works with the current images.